independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/RuntimeExceptionMapper.java

```diff
@@ -94,9 +99,17 @@ public void mapException(Throwable throwable, ResteasyReactiveRequestContext con
             } else {
                 response = exceptionMapper.toResponse(mappedException);
             }
-            context.setResult(response);
-            logBlockingErrorIfRequired(mappedException, context);
-            logNonBlockingErrorIfRequired(mappedException, context);
+            // this special case is used in order to ignore the mapping of built-in mappers and let the exception handling proceed to higher levels
+            if ((response != null) && (IGNORE_RESPONSE.getStatus() == response.getStatus())
```

Comment:
Couldn't you just test object identity here? `if (response == IGNORE_RESPONSE) { ... }`

Reply:
So simple and so true!

🙏